Jb44782

jlehtoranta opened the pull request.

martyone commented on defaults.sh:
```diff
@@ -75,7 +82,7 @@ DEF_RELCYCLE="Beta"
 
 # The default version description
-# Appears in braces after Qt Creator version in the Qt Creator About dialog
+# Appears in braces after the SDK release name in the Qt Creator and QmlLive About dialog
```
We haven't been mentioning SDK release name in the about dialogs for quite long time :) There was the issue that the compiled-in SDK release name tended to be behind the current SDK release, especially for Qt QmlLive.

martyone commented on defaults.sh:
```diff
@@ -57,15 +57,22 @@ DEF_MSVC_VER=2015
+# and QmlLive About dialog
+# Note: This can also be configured dynamically by using the -rn/--release-name
+# option in buildmaster.sh
+DEF_RELEASE_NAME="Sailfish 3 SDK"
```
See my comment in the PR to installer

jlehtoranta commented:
Updated PR:

- Dropped the release-name option
- Changed the vd/version-desc option to vp/variant-pretty and updated the helps+comments accordingly

martyone commented on buildmaster.sh:
```diff
@@ -291,14 +292,11 @@ Summary of chosen actions:
  Do Git pull on src  [$(get_option $OPT_GIT_PULL)]
  Use alt. branch ... [$OPT_BRANCH]
  SDK Config Variant  [$OPT_VARIANT]
+ SDK Release Name .. [$OPT_VARIANT_PRETTY]
```
It might be misleading that it should change with every release, while we know it is desired to remain stable. So maybe "SDK Pretty Variant" would align nicely.

jlehtoranta replied:
Updated
